cli/npm/registry.rs

```diff
@@ -350,8 +347,11 @@ impl NpmRegistryApiInner {
     }
   }
 
-  fn get_package_url(&self, name: &str) -> Url {
-    self.base_url.join(name).unwrap()
+  fn get_package_url(&self, name: &str) -> Result<Url, AnyError> {
+    match self.base_url.join(name) {
+      Ok(url) => Ok(url),
+      Err(_err) => Err(anyhow!(format!("Invalid npm package name {name:?}"))),
```

Comment:
```suggestion
      Err(_err) => Err(anyhow!(format!("Invalid npm package name '{name}'"))),
```

Prefer not using debug formats in case the type of the package name ever changes.